pkg/standalone/standalone.go

```diff
@@ -48,8 +48,63 @@ const (
 	DaprRedisContainerName = "dapr_redis"
 )
 
+func isInstallationRequired(installLocation, requestedVersion string) bool {
+	destDir := daprDefaultLinuxAndMacInstallPath
+	if installLocation == "" && runtime.GOOS == daprWindowsOS {
+		destDir = daprDefaultWindowsInstallPath
+	}
+
+	daprdBinaryPath := path_filepath.Join(destDir, daprRuntimeFilePrefix) //e.g. /usr/local/bin/daprd or c:\\daprd
```

Comment:
What if an `installLocation` is given and it's not the default path?

Reply:
> What if an `installLocation` is given and it's not the default path?

@abhirockzz This function is not using the installLocation, should it use installLocation as destDir if provided.

Reply:
destDir should suffice since it was derived from the installLocation in previous step

Reply:
> destDir should suffice since it was derived from the installLocation in previous step

@abhirockzz may be I am missing something here, but DestDir is set to default daprDefaultLinuxAndMacInstallPath on line 52.

Reply:
Got it. Thank you! Updated to handle this